bigquery/samples/delete_dataset_labels.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def delete_dataset_labels(client, dataset_id):
+
+    # [START bigquery_delete_label_dataset]
+    from google.cloud import bigquery
+
+    # TODO(developer): Construct a BigQuery client object.
+    # client = bigquery.Client()
+
+    # TODO(developer): Set dataset_id to the ID of the dataset to fetch.
+    # dataset_id = "your-project.your_dataset"
+
+    dataset = client.get_dataset(dataset_id)
+
+    # To delete a label from a dataset, set its value to None
+    dataset.labels["color"] = None
+
+    dataset = client.update_dataset(dataset, ["labels"])
+
+    dataset = client.get_dataset(dataset_id)
+
+    print("Dataset ID: {}".format(dataset_id))
```

Comment:
I wonder if we really want to be showing how to print the labels every time? It feels redundant.

Let's instead make a call to `get_dataset` in the tests for this sample and make sure the color label has been removed.